app/src/main/java/com/android/sample/model/workout/Workout.kt

```diff
@@ -11,8 +11,36 @@ abstract class Workout(
     val warmup: Boolean, // Whether the user want to do a warmup
     val userIdSet:
         MutableSet<String>, // Set of userId that represent the User linked to a specific workout
-    val date: LocalDateTime
-) {}
+    val exercises: MutableList<Exercise> = mutableListOf(),
```

Comment:
I like the way you organized the structure of this class and the three inheriting ones.

Reply:
solved